app/devicetablehandler.py

```diff
@@ -88,14 +93,24 @@ async def get(self, request):
       device_sent_dropdown_options = device_sent_dropdown('blank')
 
       return {
-          'page_title': f'Device Table view for: {user_role}',
-          'table_headers': table_headers,
-          'device_records': device_records,
-          'page_number': page_number,
-          'last_page_number': max_page,
-          'device_type_options': device_type_dropdown_options,
-          'device_sent_options': device_sent_dropdown_options,
-          'dst_download': download_permission(user_role),
+          'page_title':
+          f'Device Table view for: {user_role}',
+          'table_headers':
+          table_headers,
+          'device_records':
+          device_records,
+          'page_number':
+          page_number,
+          'last_page_number':
+          max_page,
+          'result_message':
+          result_message(search_range, device_sum, "Device Table"),
```

Comment:
Consider assigning this to a local variable like `result_message_str` so that the dictionary doesn't get split up by the formatter.

Reply:
Adjusted  as  suggested